tests/test_issues/test_pydantic_polymorphism.py

```diff
@@ -134,14 +59,21 @@ def test_pydantic_obey_range():
 
     output_subset = [line for line in output.splitlines() if "thingtype" in line]
     assert len(output_subset) > 0
-
     assert len([x for x in output_subset if "x:Person" in x]) == 1
 
     gen = PydanticGenerator(schema_str.replace("uriorcurie", "uri"))
     output = gen.serialize()
     output_subset = [line for line in output.splitlines() if "thingtype" in line]
     assert len(output_subset) > 0
     assert len([x for x in output_subset if "http://example.org/Person" in x]) == 1
+    
```

Comment:
something odd is going on with the diff - in the intervening time between creation of this PR and now we refactored to use pytest. This is showing up here like unittest except unindented...